### PR TITLE
Release notes for Node and MCO follow up

### PR DESCRIPTION
**Pull request by mburke5678**
Follow up to address @wking comment here: https://github.com/openshift/openshift-docs/pull/31237#discussion_r652271703

### CODE REVIEW

**Comment by mburke5678** [comment]
@wking @kikisdeliveryservice @mike-nguyen  PTAL

**Comment by kikisdeliveryservice** [comment]
thanks for this @mburke5678 !

@wking this is what you had in mind correct?

**Comment by mburke5678** [comment]
@mike-nguyen PTAL at this small change.

**Comment by mike-nguyen** [comment]
LGTM

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -112,7 +112,7 @@ This change does not affect machine config pool rollout duration.
 [id="ocp-4-8-mco-upgrade-complete"]
 ==== MCO waits for all machine config pools to update before reporting the update is complete
 
-When updating, the Machine Config Operator (MCO) now reports an `Upgradeable=False` status to the Cluster Version Operator (CVO) if any machine config pool has not completed updating. This status blocks future minor updates, but does not block future patch updates, or the current update. Previously, the MCO reported the `Upgradeable` status based only on the state of the master machine config pool, even if the worker pools were not done updating.
+When updating, the Machine Config Operator (MCO) now reports an `Upgradeable=False` condition in its Cluster Operator if any machine config pool has not completed updating. This status blocks future minor updates, but does not block future patch updates, or the current update. Previously, the MCO reported the `Upgradeable` status based only on the state of the master machine config pool, even if the worker pools were not done updating.
```
Comment (skrthomas) [inline release_notes/ocp-4-8-release-notes.adoc:115]: "condition in the", instead of "its"-avoid possession

**Review by skrthomas — COMMENTED** [review]
One small comment otherwise, LGTM!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (mburke5678) [reply release_notes/ocp-4-8-release-notes.adoc:115]: @skrthomas I left it as _it`s_ as each operator (I've learned) has it's own Cluster Operator. 
How about _in the machine-config Cluster Operator_ ? 
An Atom search yields a few examples, etc Cluster Operator and  console Cluster Operator. Both _etcd_ and _console_ appear to be the name of the operator (oc get clusteroperator) and do not take mark-up.

Comment (skrthomas) [reply release_notes/ocp-4-8-release-notes.adoc:115]: I like that!

**Review by adellape — APPROVED** [review]

**Pull request merged**